main.go

```diff
@@ -44,6 +45,16 @@ func main() {
 	logger.Infof("uploadable config: %+v", config.UploadableConfig)
 	logger.Infof("log config: %+v", config.LogConfig)
 
+	if config.Files == "" {
+		logger.Debug("No glob filter provided!")
```

Comment:
A few lines above, there is info message about the glob filter. Do we need to order the messages?